### PR TITLE
Avoid empty pushes from galley

### PR DESCRIPTION
**Pull request by pcapriotti**
This fixes the edge case of empty pushes being sent to gundeck from galley, due to a wrong base case in the chunk splitting function. It also adds some unit tests.

https://wearezeta.atlassian.net/browse/WPB-3169

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/galley/test/unit/Test/Galley/Intra/Push.hs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
```
Comment (mdimjasevic) [inline services/galley/test/unit/Test/Galley/Intra/Push.hs:3]:
```suggestion
-- Copyright (C) 2023 Wire Swiss GmbH <opensource@wire.com>
```

**Review by mdimjasevic — APPROVED** [review]
